src/python/tools/entrypoint_compiler.py

```diff
@@ -115,25 +115,32 @@ def decision_function(self, X, **params):
 """
 
 
-def parse_manifest(manifest_json, pkg_path=None, overwrite=False):
+def parse_manifest(manifest_json, manifest_diff_json, pkg_path=None, overwrite=False):
     """
+    Autogenerate python classes for all ML.NET entrypoints in manifest.json.
     """
     with open(manifest_json) as f:
         manifest = json.load(f)
 
+    manifest_diff = load_json(manifest_diff_json)
+
     entrypoints = manifest["EntryPoints"]  # list
+    excluded = manifest_diff["HiddenEntryPoints"] # list
```

Comment:
@shmoradims I'm interested in your feedback for how I used "HiddenEntryPoints" in manifest_diff.json.  I see that we are currently using them in a [warning message](https://github.com/Microsoft/NimbusML/blob/e1004720ec0c252ba87f02c190c33739d9c00f20/src/python/tools/manifest_diff_parser.py#L93) but I am adding this so that we can prevent codegen of entrypoints we don't want, even internally. Does this seem okay?

Also, all the current items in "HiddenEntryPoints" no longer exist in ML.NET, so I'll go ahead and delete them from the list if you're okay with that.

Reply:
This should be done through manifest.json. "HiddenEntryPoints" are just for informational purposes.

---
In reply to: [239957952](https://github.com/Microsoft/NimbusML/pull/68#discussion_r239957952) [](ancestors = 239957952)

Reply:
Thanks Shahab, that makes sense.  I'll go ahead and close this PR without checking it in, as these changes are not necessary.